### PR TITLE
Feature/add bit for code modularity

### PR DESCRIPTION
**Pull request by owodunni**

### CODE REVIEW

File: mavionics/src/modules/RtcModule.js [hunk 1]
```diff
@@ -54,42 +63,49 @@ class RtcModule {
...
   disconnect() {
-    this.p.destroy();
+    console.log("RtcModule.js sendMessdisconnectge ");
+    this.peer.destroy();
   }
 
   isConnected() {
-    return this.p && this.p.connected;
+    console.log("RtcModule.js isConnected");
+    return this.p && this.peer.connected;
```
Comment (wolkesson) [inline mavionics/src/modules/RtcModule.js]: this.p should be this.peer I assume?

File: mavionics/src/modules/RtcModule.js [hunk 2]
```diff
@@ -54,42 +63,49 @@ class RtcModule {
...
         this.dbRef.update({ status: "connected" });
         resolve();
       });
     });
   }
 
   sendMessage(data) {
-    if (this.p !== null) this.p.send(JSON.stringify(data));
+    console.log("RtcModule.js sendMessage ", JSON.stringify(data));
+    if (this.p !== null) this.peer.send(JSON.stringify(data));
```
Comment (wolkesson) [inline mavionics/src/modules/RtcModule.js]: this.p should be this.peer I assume?

**Review by wolkesson — COMMENTED** [review]
Really cool!!! Just fix the last refactoring bugs.

File: mavionics/src/modules/RtcModule.js (hunk 1 above)
Comment (owodunni) [reply mavionics/src/modules/RtcModule.js]: Could be why this has been chrashing for me ;)

Comment (owodunni) [reply mavionics/src/modules/RtcModule.js]: Could be why this has been chrashing for me ;)

**Review by wolkesson — APPROVED** [review]

**Pull request closed without merging**